R/net.mod.infection.R

```diff
@@ -178,7 +178,7 @@ infection.2g.net <- function(dat, at) {
   if (nElig > 0 && nElig < nActive) {
 
     # Get discordant edgelist
-    del <- discord_edgelist(dat, at, network = 1)
+    del <- do.call(rbind, lapply(seq_along(dat$nwparam), discord_edgelist, dat = dat, at = at, include.network = TRUE))
```

Comment:
same comment

Reply:
addressed in 17e2faf3895ebb01e215ec77b7351b204aa2d099